aws/ci_build

```diff
@@ -91,6 +95,7 @@ def main()
     commit_url = "https://github.com/code-dot-org/code-dot-org/commit/#{RakeUtils.git_revision}"
     # Notify the HipChat channels about what happened.
     projects = ARGV.join(' ')
+    projects = 'websites' if projects.empty?
```

Comment:
Can this be `projects = if !ARGV.empty? ? ARGV.join(' ') : 'websites'` or some such?